src/pages/SearchResult/SearchResult.js

```diff
@@ -1,8 +1,112 @@
+                  <div>
+                    <span>위치</span>
+                    <span>음식 카테고리</span>
+                  </div>
+                </div>
+              </span>
+              <span className="searchResultListContent">
+                <img
+                  className="searchResultMainImage"
+                  src="https://mp-seoul-image-production-s3.mangoplate.com/1760/imcyl_8.jpg?fit=around|359:240&crop=359:240;*,*&output-format=jpg&output-quality=80"
+                ></img>
+                <div className="menuTitleContainer">
+                  <span>메뉴명</span>
+                  <span>별점</span>
+                  <div>
+                    <span>위치</span>
+                    <span>음식 카테고리</span>
+                  </div>
+                </div>
+              </span>
+              <span className="searchResultListContent">
+                <img
+                  className="searchResultMainImage"
+                  src="https://mp-seoul-image-production-s3.mangoplate.com/1760/imcyl_8.jpg?fit=around|359:240&crop=359:240;*,*&output-format=jpg&output-quality=80"
+                ></img>
+              </span>
+              <span className="searchResultListContent">
+                {' '}
+                <img
+                  className="searchResultMainImage"
+                  src="https://mp-seoul-image-production-s3.mangoplate.com/1760/imcyl_8.jpg?fit=around|359:240&crop=359:240;*,*&output-format=jpg&output-quality=80"
+                ></img>
+              </span>
+              <span className="searchResultListContent">
+                {' '}
+                <img
+                  className="searchResultMainImage"
+                  src="https://mp-seoul-image-production-s3.mangoplate.com/1760/imcyl_8.jpg?fit=around|359:240&crop=359:240;*,*&output-format=jpg&output-quality=80"
+                ></img>
+              </span>
+              <span className="searchResultListContent">
+                {' '}
+                <img
+                  className="searchResultMainImage"
+                  src="https://mp-seoul-image-production-s3.mangoplate.com/1760/imcyl_8.jpg?fit=around|359:240&crop=359:240;*,*&output-format=jpg&output-quality=80"
+                ></img>
+              </span>
+              <span className="searchResultListContent">
+                {' '}
+                <img
+                  className="searchResultMainImage"
+                  src="https://mp-seoul-image-production-s3.mangoplate.com/1760/imcyl_8.jpg?fit=around|359:240&crop=359:240;*,*&output-format=jpg&output-quality=80"
+                ></img>
+              </span>
+              <span className="searchResultListContent">
+                {' '}
+                <img
+                  className="searchResultMainImage"
+                  src="https://mp-seoul-image-production-s3.mangoplate.com/1760/imcyl_8.jpg?fit=around|359:240&crop=359:240;*,*&output-format=jpg&output-quality=80"
+                ></img>
```

Comment:
```suggestion
                <img
                  alt="food"
                  className="searchResultMainImage"
                  src="https://mp-seoul-image-production-s3.mangoplate.com/1760/imcyl_8.jpg?fit=around|359:240&crop=359:240;*,*&output-format=jpg&output-quality=80"
                />
```
- img 태그에는 항상 alt 속성 넣어주세요
- 셀프 클로징 태그 적용할 수 있는 곳에는 꼭 적용해주세요